CHANGELOG.md

```diff
@@ -27,9 +27,6 @@ Each issue fixed should contain one bullet summarizing the work done.
 <hr />
 
 ## [Unreleased]
-
-
-## [[1.0.1]](https://github.com/jwu910/check-it-out/releases/tag/v1.0.1) - 2018-09-17
```

Comment:
@9reeno Regarding my last comment about merging/rebasing when you grab the latest code, make sure you are grabbing it from my remote.

The branch you worked on, should have been created from my development. and then sent back against my development. 

If you don't have my repo set up on your machine, just add mine as a remote, and you can do something like `git pull <upstream> <development>` if you named the remote `upstream`.

Let me know if you need help or have questions

Reply:
On it!

Reply:
So, if I've added your repo as a remote, I should be able to, from my local CIO-259 branch, do `git pull upstream development` and then `git merge upstream/development`?

Doing so tells me that my local branch is up to date.

Reply:
you may want to undo your last merge from your origin/development as I think that's the commit that is removing the changes from the Changelog and other files.